common-api/src/main/java/org/pmiops/workbench/config/WorkbenchConfig.java

```diff
@@ -151,5 +151,7 @@ public static WorkbenchConfig createEmptyConfig() {
     // Whether or not AoU should request Terra to create GCP projects inside a VPC
     // security perimeter.
     public boolean enableVpcServicePerimeter;
+    // Flag to indicate if the new create account as per CAPS requirement
```

Comment:
"Flag to indicate whether to enable the new Create Account flow" and add relevant JIRA stories